code/nodes/optimizeModels/utils.q

```diff
@@ -1,22 +1,28 @@
+// code/nodes/optimizeModels/utils.q - Utilities for the optimizeModels node
+// Copyright (c) 2021 Kx Systems Inc
+//
+// Utility functions specific the the optimizeModels node implementation
+
 \d .automl
 
 // Utility functions for optimizeModels
 
 // @kind function
 // @category optimizeModelsUtility
-// @fileoverview Extract the hyperparameter dictionaries based on the applied model
+// @desc Extract the hyperparameter dictionaries based on the applied model
 // @param bestModel  {<} Fitted best Model
-// @param cfg        {dict} Configuration information assigned 
+// @param cfg        {dictionary} Configuration information assigned 
```

Comment:
Extra space between param name and type can be removed for all the below params in the file